ctapipe_io_magic/__init__.py

```diff
@@ -1124,31 +1127,30 @@ def _event_generator(self):
                 event_data = self.current_run['data'].cosmic_events
                 n_events = self.current_run['data'].n_cosmic_events
 
-            if not self.is_simulation:
-
-                monitoring_data = self.current_run['data'].monitoring_data
+            monitoring_data = self.current_run['data'].monitoring_data
 
-                # Set the pedestal information:
-                event.mon.tel[tel_id].pedestal.n_events = 500   # hardcoded number of pedestal events averaged over
-                event.mon.tel[tel_id].pedestal.sample_time = monitoring_data['pedestal_sample_time']
+            # Set the pedestal information:
+            event.mon.tel[tel_id].pedestal.n_events = 500   # hardcoded number of pedestal events averaged over
```

Comment:
you can add a note that this is valid for pedestal_from_extractor{_rndm}, the fundamental one is updated every 100 events